qemu/tests/virtio_port_hotplug.py

```diff
@@ -68,20 +80,21 @@ def run(test, params, env):
                 time.sleep(0.5)
                 vm.devices.simple_hotplug(port_chardev, vm.monitor)
             vm.devices.simple_hotplug(virtio_port, vm.monitor)
-            if module:
+            if module and check_module:
                 error_context.context("Load  module %s" % module, logging.info)
                 session.cmd("modprobe %s" % module)
+                time.sleep(1)
         session.close()
-    host_script = params['host_script']
-    check_pid_cmd = 'pgrep -f %s' % host_script
-    host_proc_pid = process.getoutput(check_pid_cmd, shell=True)
-    if host_proc_pid:
-        logging.info("Kill the first serial process on host")
-        result = process.system('kill -9 %s' % host_proc_pid, shell=True)
-        if result != 0:
-            logging.error("Failed to kill the first serial process on host!")
-    if transfer_data(params, vm) is not True:
-        test.fail("Serial data transfter test failed.")
+        host_script = params['host_script']
+        check_pid_cmd = 'pgrep -f %s'
+        host_proc_pid = process.getoutput(check_pid_cmd % host_script, shell=True)
+        if host_proc_pid:
+            logging.info("Kill the first serial process on host")
+            result = process.system('kill -9 %s' % host_proc_pid, shell=True)
+            if result != 0:
+                logging.error("Failed to kill the first serial process on host!")
+        if transfer_data(params, vm) is not True:
+            test.fail("Serial data transfter test failed.")
```

Comment:
There are space or tab in these code, pls remove them, thanks.

Others looks good to me.
 (1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.virtio_port_hotplug.q35: PASS (623.82 s)

Reply:
> There are space or tab in these code, pls remove them, thanks.
> 
> Others looks good to me.
> (1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.virtio_port_hotplug.q35: PASS (623.82 s)

Thanks for testing, I move the check step into the loop since we need to test two virtserialports.

Reply:
Get it.